cmd/juju/status/output_tabular.go

```diff
@@ -155,13 +155,23 @@ func printApplications(tw *ansiterm.TabWriter, fs formattedStatus) {
 		} else {
 			w.Print(scale)
 		}
+
 		w.Print(app.CharmName,
 			app.CharmOrigin,
 			app.CharmRev,
 			app.OS)
 		if fs.Model.Type == caasModelType {
 			w.Print(app.Address)
 		}
+
+		var charmVersion string
```

Comment:
Easiest way to truncate a string is like this:

w.Print(fmt.Printf("%.15s", a.CharmVersion))

Reply:
Don't we have to check for the case if the file is empty ?

Reply:
No the Printf will work just fine even with an empty string